eo-maven-plugin/src/main/java/org/eolang/maven/Moja.java

```diff
@@ -147,30 +146,35 @@ private static Collection<Field> fields(final Class<?> cls) {
     }
 
     /**
-     * Take a field.
-     * @param mojo The class
-     * @param name Field name
-     * @return Field
+     * Init a field.
+     * @param clazz The mojo class
+     * @param mojo The mojo
+     * @param entry Field name and value
+     * @throws java.lang.IllegalAccessException If can't set field.
      */
-    private Field field(final Class<?> mojo, final String name) {
-        Field field;
+    private void initField(
+        final Class<?> clazz,
+        final AbstractMojo mojo,
+        final Map.Entry<String, Object> entry
+    ) throws IllegalAccessException {
+        final String name = entry.getKey();
         try {
-            field = mojo.getDeclaredField(name);
+            final Field field = clazz.getDeclaredField(name);
+            field.setAccessible(true);
+            field.set(mojo, entry.getValue());
         } catch (final NoSuchFieldException ex) {
```

Comment:
@volodya-lombrozo here we use exception for control flow. Can we use `clazz.getDecleardFields()` instead to get rid of `try-catch` block?

Reply:
@mximp fixed. Please, check.